### PR TITLE
[AIRFLOW-3637] Fix test for HiveToMySqlTransfer Operator

### PR DESCRIPTION
**Pull request by feng-tao**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-3637
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
Fix a test in TestHiveToMySqlTransfer introduced in https://github.com/apache/airflow/pull/4432 due to existing CI issue.

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by feng-tao** [comment]
This should fix test_execute (tests.operators.test_hive_to_mysql.TestHiveToMySqlTransfer) ... FAILED

**Comment by feng-tao** [comment]
PTAL @kaxil , once I confirm the CI for this test passes, I will commit the change.

**Comment by feng-tao** [comment]
Confirm that all the tests under ``TestHiveToMySqlTransfer`` is fixed. Commit this change now.

**Comment by feng-tao** [comment]
FYI @kaxil @aliceabe

File: tests/operators/test_hive_to_mysql.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def test_execute(self, mock_hive_hook, mock_mysql_hook):
         HiveToMySqlTransfer(**self.kwargs).execute(context={})
 
         mock_hive_hook.assert_called_once_with(hiveserver2_conn_id=self.kwargs['hiveserver2_conn_id'])
-        mock_hive_hook.return_value.get_records.assert_called_once_with(self.kwargs['sql'])
+        mock_hive_hook.return_value.get_records.assert_called_once_with('sql', hive_conf={})
```
Comment (feluelle) [inline tests/operators/test_hive_to_mysql.py:46]: Why did you change the first param to a static value instead of the reference that is being passed to the `HiveToMySqlTransfer` constructor? 
I mean why not `self.kwargs['sql'], hive_conf={}`. This way it would even work if you change the kwargs input dict.

**Comment by aliceabe** [comment]
Thank you @feng-tao
